### ISSUE TITLE
[Login] Inputs missing autocomplete attribute

### ISSUE THREAD

**Issue by svinkle** [CovidShield/portal#4]
## Issue summary

The login `input` fields are missing the [`autocomplete` attribute](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofilling-form-controls:-the-autocomplete-attribute). This helps users of assistive technology to complete filling in forms by using data stored in the browser.

<img width="382" alt="Screen Shot 2020-05-19 at 10 13 50 AM" src="https://user-images.githubusercontent.com/1392632/82337191-7ccc8880-99b9-11ea-8679-86b8c57ab0ef.png">

#### HTML

```html
<input
  type="text"
  name="username"
  id="username"
  …
/>

<!-- … -->

<input
  type="password"
  name="password"
  id="password"
  …
/>
```

## Steps to reproduce

1. Go to https://portal.covidshield.app/ with with any supported browser
2. Use the inspector to review the markup of the `input` fields

### Expected

- Field to include the `autocomplete` attribute.

### Actual

- The `autocomplete` attribute is not present.

## Recommendation

1. Add the `autocomplete` attribute to the Username `input`, setting its value to `username`.
1. Add the `autocomplete` attribute to the Password `input`, setting its value to `current-password`.

#### HTML

```html
<input
  type="text"
  name="username"
  id="username"
  …
  autocomplete="username"
/>

<!-- … -->

<input
  type="password"
  name="password"
  id="password"
  …
  autocomplete="current-password"
/>
```

## Specifications

- **Component:** Login
- **WCAG Principle:** Perceivable
- **WCAG SC:** 1.3.5 Identify Input Purpose
- **Severity:** Medium
- **Effort:** Low
